menu/menu_setting.c

```diff
@@ -15380,6 +15380,22 @@ static bool setting_append_list(
                SD_FLAG_NONE
                );
 
+         CONFIG_BOOL(
+               list, list_info,
+               &settings->bools.cheevos_richpresence_enable,
+               MENU_ENUM_LABEL_CHEEVOS_RICHPRESENCE_ENABLE,
+               MENU_ENUM_LABEL_VALUE_CHEEVOS_RICHPRESENCE_ENABLE,
+               false,
```

Comment:
Something to do with reset behavior and hardcore; I wanted it to only ever work when hardcore was active and some LBs could start when a save was autoloaded. Too long ago for me to remember

For this I defaulted to OFF because the Discord privacy setting does as well